internal/pkg/cli/app_init.go

```diff
@@ -190,18 +181,33 @@ func (o *initAppOpts) createManifest() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	var manifestExist bool
 	manifestPath, err := o.ws.WriteAppManifest(manifest, o.AppName)
 	if err != nil {
-		return "", err
+		var fileExistsErr *workspace.ErrFileExists
+		if !errors.As(err, &fileExistsErr) {
+			return "", err
+		}
+		manifestExist = true
 	}
 	wkdir, err := os.Getwd()
 	if err != nil {
 		return "", fmt.Errorf("get working directory: %w", err)
 	}
 	relPath, err := filepath.Rel(wkdir, manifestPath)
 	if err != nil {
-		return "", fmt.Errorf("relative path of manifest file: %w", err)
+		return "", fmt.Errorf("get relative path of manifest file: %w", err)
+	}
+
+	log.Infoln()
+	if manifestExist {
+		log.Successf("Skip writing since the manifest for %s app already exists at %s\n", color.HighlightUserInput(o.AppName), color.HighlightResource(relPath))
+	} else {
+		log.Successf("Wrote the manifest for %s app at %s\n", color.HighlightUserInput(o.AppName), color.HighlightResource(relPath))
```

Comment:
nit:
```go
manifestMsgFmt := "Wrote the manifest for %s app at %s\n"
if manifestExists {
     manifestMsgFmt = "Manifest file for %s app already exists at %s, skipping writing it.\n"
}
log.Successf(manifestMsgFmt, color.HighlightUserInput(o.AppName), color.HighlightResource(relPath))
```
In general, the moment we see an `else` statement that's a hint that we can possibly restructure the code and get rid of it :)